### ISSUE TITLE
Non-square Images?

### ISSUE THREAD

**Issue by researchhappy** [NVlabs/stylegan2-ada-pytorch#289]
Simple Question: Is there a way to input non-square and non-rectangular images? For example, if I am training the model on an apple, I want to only crop out the apple (no padding, no nothing) and train the model on only that particular part of the generation.

**Comment by Prasantkumar987** [comment]
@researchhappy Yes you can input rectangular inputs  but you have specify `width` and `height` parameter while preparing dataset using `dataset_tool.py`.

There is another way while using `dataset_tool.py` for data preperation set `--cropsize`  parameter
